packages/validator/src/validator.ts

```diff
@@ -159,6 +162,32 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        let feeRecipient = "";
+
+        try {
+          feeRecipient = validatorStore.getFeeRecipient(pubkeyHex);
```

Comment:
do we really expect that this can throw an error? we are iterating over the validator keys which we got from `validatorStore` and `getFeeRecipient` only throw an error if pubkey does not exist

https://github.com/ChainSafe/lodestar/blob/df1ef97e9050a588b74f21bfae314d675caea288/packages/validator/src/services/validatorStore.ts#L184-L186

Reply:
See https://github.com/ChainSafe/lodestar/pull/5239#discussion_r1147803452